[2022-06-18 05:14:56,833] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.my_etl manual__2022-06-18T05:14:55.788395+00:00 [queued]>
[2022-06-18 05:14:56,844] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.my_etl manual__2022-06-18T05:14:55.788395+00:00 [queued]>
[2022-06-18 05:14:56,844] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-18 05:14:56,844] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-18 05:14:56,844] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-18 05:14:56,939] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): my_etl> on 2022-06-18 05:14:55.788395+00:00
[2022-06-18 05:14:56,948] {standard_task_runner.py:52} INFO - Started process 118 to run task
[2022-06-18 05:14:56,952] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test', 'my_etl', 'manual__2022-06-18T05:14:55.788395+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/test-dag.py', '--cfg-path', '/tmp/tmpwg7wvlhk', '--error-file', '/tmp/tmpyrogo7th']
[2022-06-18 05:14:56,953] {standard_task_runner.py:80} INFO - Job 137: Subtask my_etl
[2022-06-18 05:14:57,030] {task_command.py:370} INFO - Running <TaskInstance: test.my_etl manual__2022-06-18T05:14:55.788395+00:00 [running]> on host 98ef215c91ca
[2022-06-18 05:14:57,139] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=my_etl
AIRFLOW_CTX_EXECUTION_DATE=2022-06-18T05:14:55.788395+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-18T05:14:55.788395+00:00
[2022-06-18 05:14:57,140] {logging_mixin.py:115} INFO - Hello there
[2022-06-18 05:14:57,140] {python.py:173} INFO - Done. Returned value was: None
[2022-06-18 05:14:57,162] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test, task_id=my_etl, execution_date=20220618T051455, start_date=20220618T051456, end_date=20220618T051457
[2022-06-18 05:14:57,203] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-18 05:14:57,263] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
